zeebe/backup-stores/s3/src/main/java/io/camunda/zeebe/backup/s3/FileSetManager.java

```diff
@@ -53,50 +54,62 @@ public FileSetManager(final S3AsyncClient client, final S3BackupConfig config) {
     // concurrent uploads to half of the number of available connections.
     // This should prevent ConnectionAcquisitionTimeout for backups with many and/or large files
     // where we would otherwise occupy all connections, preventing some uploads from starting.
-    uploadLimit = new Semaphore(Math.max(1, config.maxConcurrentConnections() / 2));
+    concurrencyLimit = new Semaphore(Math.max(1, config.maxConcurrentConnections() / 2));
+    threadBuilder = Thread.ofVirtual().name("zeebe-backup-", 0);
```

Comment:
```suggestion
    threadBuilder = Thread.ofVirtual().name("zeebe-backup-s3", 0);
```
So it's clear it's the `S3BackupStore` and not for example the `BackupService`?